src/AppInstallerCommonCore/AppInstallerTelemetry.cpp

```diff
@@ -11,6 +11,15 @@
 #define AICLI_TraceLoggingStringView(_sv_,_name_) TraceLoggingCountedUtf8String(_sv_.data(), static_cast<ULONG>(_sv_.size()), _name_)
 #define AICLI_TraceLoggingWStringView(_sv_,_name_) TraceLoggingCountedWideString(_sv_.data(), static_cast<ULONG>(_sv_.size()), _name_)
 
+#define AICLI_TraceLoggingWriteActivity(_eventName_,...) TraceLoggingWriteActivity(\
+g_hTraceProvider,\
+_eventName_,\
+GetActivityId(false),\
+nullptr,\
+TraceLoggingCountedString(m_caller.c_str(),  static_cast<ULONG>(m_caller.size()), "Caller"),\
+TraceLoggingPackedFieldEx(GetTelemetryCorelationJsonW().c_str(), static_cast<ULONG>((GetTelemetryCorelationJsonW().size() + 1) * sizeof(wchar_t)), TlgInUNICODESTRING, TlgOutJSON, "CvJson"),\
```

Comment:
This should use the member, or the function should be changed to return a `const &`.

Reply:
Used member instead.